shared-kcp/create-workload-cluster.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+set -o nounset
+set -o pipefail
+set -o errexit
+
+if ! which jq &> /dev/null; then
+    echo "Install jq"
+    exit 1
+fi
+
+if [ $# -eq 0 ]; then
```

Comment:
You can look at gitops.sh for parsing args, and showing the `usage` with `--help`.

Reply:
This is not our script. Have taken this from the onboarding doc. Have added this point to README.
https://gitlab.cee.redhat.com/kcp/shared-resources/-/blob/main/scripts/create-workload-cluster.sh